jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -803,28 +806,29 @@ public Action process() throws Exception
                             gatherWrite += 2;
                             bytes += _chunk.remaining();
                         }
-                        if (BufferUtil.hasContent(_content))
+                        long contentLength = getRemainingContent();
```

Comment:
Isn't this going to be the same length as calculated above?

Reply:
I was getting failures from calculating this only once in the constructor so I changed to calculate wherever used to be safe.

But it seems to pass tests if I just calculate once at the start of `process()`, so I will try doing that instead.